### PR TITLE
Clean version of timer docs

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

This adds the new app-dev guide generation script to the Timer docs.

[Completes the feedback from the previous PR.](https://github.com/temporalio/documentation/pull/1493)

## Notes to reviewers
SDK team to review the following code snippets.
- [ ] Go
- [ ] Java
- [ ] PHP
- [ ] Python
- [ ] TypeScript

### CODE REVIEW

File: docs-src/python/how-to-set-timers-in-python.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+id: how-to-set-timers-in-python
+title: How to set Timers in Python
+sidebar_label: Python
+description: To set a Timer in Python, set the `asyncio.sleep()` function.
+tags:
+  - timers
+  - sleep
+---
+
+To set a Timer in Python, set the [`asyncio.sleep()`](https://docs.python.org/3/library/asyncio-task.html#sleeping) function and pass the duration you want to wait in seconds before continuing.
```
Comment (djmagee) [inline docs-src/python/how-to-set-timers-in-python.md:11]:
```suggestion
To set a Timer in Python, set the [`asyncio.sleep()`](https://docs.python.org/3/library/asyncio-task.html#sleeping) function and pass the duration in seconds you want to wait before continuing.
```
Or "...pass the number of seconds you want to wait before continuing" to match the text in other SDKs.

**Review by djmagee — APPROVED** [review]
Just one edit.

**Review by djmagee — APPROVED** [review]

File: docs-src/app-dev-context/timers.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - timers
+  - sleep
+---
+
+A Workflow can set a durable timer for a fixed time period.
+In some SDKs, the function is called `timer()`, and in others, it's called `sleep()`.
+
+A Workflow can sleep for months.
+Even if your Worker crashes or your Temporal Cluster goes down, Timers are persisted and will fire as set.
+As soon as your Worker and Cluster are running, your code resumes where it left off.
```
Comment (lorensr) [inline docs-src/app-dev-context/timers.md:17]:
```suggestion
Timers are persisted, so even if your Worker or Temporal Cluster is down when the time period completes, as soon as your Worker and Cluster are back up, the `sleep()` call will resolve and your code will continue executing.
```

File: docs-src/app-dev-context/timers.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+description: A Timer lets a Workflow sleep for a fixed time period.
+sidebar_label: Timers
+tags:
+  - guide-context
+  - timers
+  - sleep
+---
+
+A Workflow can set a durable timer for a fixed time period.
+In some SDKs, the function is called `timer()`, and in others, it's called `sleep()`.
```
Comment (lorensr) [inline docs-src/app-dev-context/timers.md:13]:
```suggestion
In some SDKs, the function is called `sleep()`, and in others, it's called `timer()`.
```

Looks like sleep is more common, so putting first

File: docs-src/python/how-to-set-timers-in-python.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+id: how-to-set-timers-in-python
+title: How to set Timers in Python
+sidebar_label: Python
+description: To set a Timer in Python, set the `asyncio.sleep()` function.
+tags:
+  - timers
+  - sleep
+---
+
+To set a Timer in Python, set the [`asyncio.sleep()`](https://docs.python.org/3/library/asyncio-task.html#sleeping) function and pass the duration in seconds you want to wait before continuing.
```
Comment (lorensr) [inline docs-src/python/how-to-set-timers-in-python.md:11]:
```suggestion
To set a Timer in Python, call the [`asyncio.sleep()`](https://docs.python.org/3/library/asyncio-task.html#sleeping) function and pass the duration in seconds you want to wait before continuing.
```

**Review by lorensr — APPROVED** [review]

**Pull request merged**